ci-scripts/runs-to-csv.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Just a helper script to output CSV file based on all found benchmark-tekton.json files
+
+find . -name benchmark.json -print0 | while IFS= read -r -d '' filename; do 
+    jq <"${filename}" --raw-output '[
```

Comment:
@jhutar Please apply [this](https://github.com/redhat-performance/backstage-performance/blob/main/ci-scripts/scalability/collect-results.sh#L76) filtering for `benchmark.json` that fixes some unexpected censoring (with `XXXXXXX`) that OpenShift CI does when gathering artifacts.

Reply:
Nice! Added.